apps/web/src/services/alert-manager/alert-dashboard/modules/alert-history-widget/AlertHistoryChart.vue

```diff
@@ -196,7 +159,8 @@ const getDailyAlertHistory = async () => {
             end: props.currentDate.add(1, 'month').format('YYYY-MM-01'),
             activated_projects: props.activatedProjects,
         });
-        state.chartData = initChartData(results);
+        if (results.length) state.chartData = initChartData(results);
+        else state.chartData = [];
```

Comment:
Suggestion: This approach can lead to issues such as a blank array being provided every time an API call is canceled, causing flickering, among other problems. From the user's perspective, when an API call is canceled, it's as if nothing happened. Therefore, it might be better to avoid making any changes in such cases.